src/components/Home.js

```diff
@@ -10,7 +10,7 @@ const Home = () => {
   const [joinlist, setJoinList] = useState('');
   const [noSharedToken, setNoSharedToken] = useState(false);
   const [showErrorMessage, setShowErrorMessage] = useState(true);
-  const [token, setToken] = useState(() => {
+  const [token] = useState(() => {
     const existingToken = localStorage.getItem('currToken');
     return existingToken ? existingToken : '';
   });
```

Comment:
the same thing I said about useState above, if you don't need the `setState` function then you don't need a useState hook. In this case, because it is an asynchronous function you could use a try-catch block as soon as the component mounts.